core/commands/dns.go

```diff
@@ -26,7 +26,7 @@ This command resolves those links to the referenced object.
 
 For example, with this DNS TXT record:
 
-	> dig +short TXT ipfs.io
+	> dig +short TXT _dnslink.ipfs.io
```

Comment:
Working example is much better than dummy one, `example.com` is nice because it works, you can fetch A record of it, you can open it via HTTP, but it won't work in this case.